example/index.scss

```diff
@@ -1,6 +1,6 @@
-@import "@edx/brand/paragon/fonts";
-@import "@edx/brand/paragon/variables";
-@import "@edx/paragon/scss/core/core";
-@import "@edx/brand/paragon/overrides";
+@import url("https://cdn.jsdelivr.net/npm/@edx/brand-edx.org@alpha/dist/core.min.css");
+@import url("https://cdn.jsdelivr.net/npm/@edx/brand-edx.org@alpha/dist/light.min.css");
+@import "@edx/paragon/styles/scss/core/core";
+@import "@edx/paragon/styles/css/themes/light/index.css";
```

Comment:
[clarification] As is, I believe these would be importing duplicate styles? For example, `@import "@edx/paragon/styles/css/themes/light/index.css";` is largely the same import as `@import url("https://cdn.jsdelivr.net/npm/@edx/brand-edx.org@alpha/dist/light.min.css");` above.

If we're importing the `core.css` and `light.css` files via `example/index.scss`, we should likely rely on the local imports from `@edx/paragon`. Take a look at the draft migration documentation for an example of what I imagine MFEs would need to do to import the compiled Paragon CSS locally: https://openedx.atlassian.net/wiki/spaces/BPL/pages/3770744958/Migrating+MFEs+to+Paragon+design+tokens+and+CSS+variables#Using-locally-installed-Paragon-CSS

Alternatively, we could also consider configuring the example MFE to utilize the `@edx/frontend-platform` approach to injecting the Paragon CSS from external CDN url like jsDelivr once https://github.com/openedx/frontend-platform/pull/440 merges.